### PR TITLE
Clean up page container widths demonstration and show code example

### PR DESCRIPTION
**Pull request by halocline**
[Deploy Preview](https://deploy-preview-2403--keen-mayer-a86c8b.netlify.app/templates/page-layouts#page-container-widths-demonstration)

#### What does this PR do?

Clean up page container widths demonstration and show code example

#### Where should the reviewer start?

/examples/templates/page-layouts/PageContainerInteractive.js

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [x] Small, medium, and large screen sizes
- [x] Cross-browsers (FireFox, Chrome, and Safari)
- [x] Light & dark modes
- [x] All hyperlinks route properly

**Accessibility Checks**
- [x] Keyboard interactions
- [ ] Screen reader experience
- [x] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [x] Console is free of warnings and errors
- [x] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

#### Any background context you want to provide?

https://grommet.slack.com/archives/C09QQEG69/p1646062195976269?thread_ts=1645989947.125189&cid=C09QQEG69

### CODE REVIEW

File: aries-site/src/examples/templates/page-layouts/PageContainerInteractive.js [hunk 1]
```diff
@@ -21,25 +21,24 @@ const demoStyle = {
...
+    //  Remove from your implementation.
     as={ContentArea}
     title="AppContainer"
     {...demoStyle}
   >
     {/* ContentArea is specific for the Design System site and is used
-     * as a visual aid to help present layout concepts. Remove from your
-     * implementation.
-     */}
+     as a visual aid to help present layout concepts. Remove from your
```
Comment (britt6612) [inline aries-site/src/examples/templates/page-layouts/PageContainerInteractive.js:31]: should we add `Content Area` just to not get confused

**Review by britt6612 — APPROVED** [review]
Just one comment otherwise looks good

File: aries-site/src/examples/templates/page-layouts/PageContainerInteractive.js (hunk 1 above)
Comment (halocline) [reply aries-site/src/examples/templates/page-layouts/PageContainerInteractive.js]: Done.

File: aries-site/src/examples/templates/page-layouts/PageContainerInteractive.js [hunk 2]
```diff
@@ -21,25 +21,24 @@ const demoStyle = {
...
   >
     {/* ContentArea is specific for the Design System site and is used
-     * as a visual aid to help present layout concepts. Remove from your
-     * implementation.
-     */}
+     as a visual aid to help present layout concepts. Remove ContentAreas from
+     your implementation. */}
     <ContentArea title="Global Header" background="status-unknown" />
+    {/* PageContainer is a context provider which specifies the page's 
+      max-width, margins, and horizontal alignment for it's children.
```
Comment (taysea) [inline aries-site/src/examples/templates/page-layouts/PageContainerInteractive.js:35]:
```suggestion
      max-width, margins, and horizontal alignment for its children. 
```

**Review by taysea — APPROVED** [review]

**Pull request merged**